### DISCUSSION TITLE
The "license" key from PyPI info is empty?

### DISCUSSION THREAD

**Discussion by lilydjwg** [encode/httpx#2454] [Potential Issue]
In [version 0.23.0](https://pypi.org/pypi/httpx/0.23.0/json) and before there is a "license" key with value "BSD" but in [version 0.23.1](https://pypi.org/pypi/httpx/0.23.1/json) it's empty. Also the "home_page" key has become empty too (though there is still "project_urls.HomePage").

Does this happen by mistake during the switch to the new packaging tool or is it something intentionally changed (deprecated? I haven't found explanations for the returned JSON anyway)?

**Comment by lovelydinosaur** [comment]
Not sure what's up with that. Not aware of anything we've changed anything in our PyPI deployment that'd cause the change.

**Reply by Kludex** [reply]
cc @ofek

**Reply by ofek** [reply]
https://github.com/squidfunk/mkdocs-material/pull/4392#issuecomment-1261352190
